src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -604,6 +605,18 @@ function ChildReconciler(shouldClone, shouldTrackSideEffects) {
         case REACT_ELEMENT_TYPE:
         case REACT_COROUTINE_TYPE:
         case REACT_PORTAL_TYPE:
+          const getMainExplicitKeyWarning = () => {
+            const message = 'Each child in an array or iterator should have a unique ' +
+              '"key" prop.' +
+              'See https://fb.me/react-warning-keys for more information.';
+            return message;
+          };
```

Comment:
Maybe just return the string from the arrow function directly?

```js
const getMainExplicitKeyWarning = () => (
 'Each child in an array or iterator should have a unique ' +
 '"key" prop.' +
 'See https://fb.me/react-warning-keys for more information.';
)
```

This could also be moved out of `warnOnDuplicateKey`  into it's own DEV block too